hathor/event/event_manager.py

```diff
@@ -209,20 +221,24 @@ def _create_event(
             group_id=group_id,
         )
 
-    def _handle_load_started(self):
+    def _handle_load_started(self) -> None:
+        """Event specific handler for EventType.LOAD_STARTED."""
         self._event_storage.save_node_state(NodeState.LOAD)
 
-    def _handle_load_finished(self):
+    def _handle_load_finished(self) -> None:
+        """Event specific handler for EventType.LOAD_FINISHED."""
         self._event_storage.save_node_state(NodeState.SYNC)
 
     def _should_reload_events(self) -> bool:
+        """Returns whether events should be reloaded or not."""
         return self._previous_node_state in [None, NodeState.LOAD]
 
     def get_event_queue_state(self) -> bool:
-        """Get whether the event queue feature is enabled from the storage"""
+        """Gets whether the event queue feature is enabled from the storage."""
```

Comment:
Convention is to use the imperative on docstring. So, `Get whether ... `.

Reply:
Really? I always thought it was the opposite, there's a lot of code using the descriptive version, just search for "returns"... But I changed this in https://github.com/HathorNetwork/hathor-core/pull/647/commits/99cf418bb73591c67e42fd0520e7cd2ac94288e8

Reply:
Our docstrings are far from the convention. Anyway, thanks for changing.